packages/Material-Tools.package/MDThemeConfigurationWindow.class/instance/buildBrightnessButtons.st

```diff
@@ -0,0 +1,33 @@
+initialization
+buildBrightnessButtons
+
+	| wrapper lightTheme darkTheme |
+	wrapper := Morph new
+		color: Color transparent;
+		layoutPolicy: TableLayout new;
+		cellPositioning: #topLeft;
+		listDirection: #topToBottom;
+		yourself.
+
+	lightTheme := MDOutlinedButton new
+		hResizing: #spaceFill;
+		text: 'Light theme';
+		addClickedActionBlock: [Transcript showln: 'light'];
+		"MDColorTheme currentTheme switchToLightTheme"
```

Comment:
I assume this line is commented out because theme switching is not supported yet? In that case we should probably wait with merging until that feature has been merged.

Reply:
Yes, that's for #189. I'd prefer to merge this PR already and just swap the `addClickedActionBlock:` argument when dark theme is ready, though. I'll make sure we don't forget it in the respective PR